### DISCUSSION TITLE
Add an option to disable `child_process` patching

### DISCUSSION THREAD

**Discussion by viceice** [yao-pkg/pkg#26] [Ideas]
We don't want that our compiled binary behaves as node, so we like to have a buildin method to disable the `child_process` patching which is solving it for our needs.

We currently use a pnpm patch to achive this, but it needs to be adapted manually for each new pkg version

**Comment by robertsLando** [comment]
I think this should work with an env var, would you like to submit a PR? Remember to document it then

**Reply by viceice** [reply]
sure, but that env var needs to be compiled into the code, i can't ensure it at runtime. 🤔

**Reply by robertsLando** [reply]
@viceice What do you mean? If you want to "bake" the env in your exe Just set the env in your entrypoint and you are done

**Reply by viceice** [reply]
ok, will try it with a local patch bevore submitting a PR. 🤗
